### PR TITLE
[AKS] GA Kubernetes version alias

### PR DESCRIPTION
**Pull request by zjpjack**
**Related command**

az aks create --kubernetes-version
az aks show

**Description**

(1) AKS supports ManagedCluster kubernetes version with format major.minor, so remove the original limitation version number must have patch number
(2) AKS has both kubernetesVersion and currentKubernetesVersion, currentKubernetesVersion is the actual version that MC running. We add it in the az aks show command

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

**Review by FumingZhang — APPROVED** [review]
LGTM

File: src/azure-cli/azure/cli/command_modules/acs/_validators.py [hunk 1]
```diff
@@ -94,13 +94,13 @@ def validate_k8s_version(namespace):
     """Validates a string as a possible Kubernetes version. An empty string is also valid, which tells the server
     to use its default version."""
     if namespace.kubernetes_version:
-        k8s_release_regex = re.compile(r'^[v|V]?(\d+\.\d+\.\d+.*)$')
```
Comment (moreOver0) [inline src/azure-cli/azure/cli/command_modules/acs/_validators.py:97]: Is reducing the `.*` expected?

Comment (zjpjack) [reply src/azure-cli/azure/cli/command_modules/acs/_validators.py:97]: yes, this is expected. Actually, the original regex has some issues. With .*, some version like 1.2.3.4 is allowed, but we only allow major.minor.patch, which is 1.2.3 is allowed. And our new regex will allow both 1.2 or 1.2.3, but 1.2.3.4 is not allowed.

**Review by moreOver0 — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**